src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -724,7 +724,9 @@ private void effectTick(long time) {
         final List<TimedPotion> effects = this.effects;
         if (effects.isEmpty()) return;
         effects.removeIf(timedPotion -> {
-            final long potionTime = (long) timedPotion.getPotion().duration() * MinecraftServer.TICK_MS;
+            long duration = timedPotion.getPotion().duration();
+            if (duration == Potion.INFINITE_DURATION) return false;
+            final long potionTime = duration * MinecraftServer.TICK_MS;
```

Comment:
Add a space here, just my opinion.

Reply:
No, there should not be an extra space on this line